R/Auth.R

```diff
@@ -0,0 +1,75 @@
+#' Create a connection to an instance of WebAPI
+#' 
+#' This function creates a WebAPI connection object that can be used by other
+#' functions in this package to access WebAPI. This function will test 
+#' the connection as well as trigger authentication and store the auth
+#' session token.
+#'
+#' @template baseUrl 
+#' @param authMethod The method used for authentication to WebAPI. Options are
```

Comment:
Recommend adding (maybe checkmate) that checks if the user has chosen from possible choices